src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
```

Comment:
What is 40? Let's create a variable for this value to make your code more understandable

Reply:
not fixed. Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content. 
`lowDiscount` - good
`highDiscount` - good
`????😢 ` - means 40 create variable for this number